src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../../TodosContext/TodosContext';
+import { FilteringBy } from '../../types/FilteringBy';
+
+export const Footer: React.FC = () => {
+  const { todos, handleClearCompleted, filteringBy, setFilteringBy } =
+    useContext(TodosContext);
+
+  const [countOfComplitedTodos, setCountOfComplitedTodos] = useState(0);
+  const [isAnyCompletedTodo, setIsAnyCompletedTodo] = useState(false);
+
+  useEffect(() => {
+    const complitedTodos = todos.filter(todo => !todo.completed).length;
+
+    setCountOfComplitedTodos(complitedTodos);
+
+    const isAnyCompleted = todos.some(todo => todo.completed);
+
+    setIsAnyCompletedTodo(isAnyCompleted);
+  }, [todos]);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${countOfComplitedTodos} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filteringBy === FilteringBy.default,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilteringBy(FilteringBy.default)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filteringBy === FilteringBy.active,
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilteringBy(FilteringBy.active)}
+        >
+          Active
+        </a>
```

Comment:
You can get an array of values from your enum with `Object.values(FilteringBy)` and then map through them and render